lib/elixir/lib/uri.ex

```diff
@@ -73,42 +73,73 @@ defmodule URI do
   @doc """
   Encodes an enumerable into a query string.
 
+  This function is deprecated. Use `encode_query/2` with `:www_form` as second
+  argument value to achieve the same result.
+  """
+  @doc deprecated: "Use URI.encode_query/2 instead"
+  @spec encode_query(Enum.t()) :: binary
+  def encode_query(enumerable) do
+    encode_query(enumerable, :www_form)
+  end
```

Comment:
Why deprecate, and not just make `:www_form` the default when called as a 1-arity function?

Reply:
Well, what you propose is what was my original proposal. Deprecation was suggested by someone else and this suggestion was seen as an improvement by @josevalim, specifically in this message:

https://groups.google.com/g/elixir-lang-core/c/gMqIFBr4OTU/m/uue1MSulDQAJ

I'm not specifically for of against deprecation, I'm happy to leave this decision to the Elixir core team.

Reply:
Ok, so no deprecation it is, per https://github.com/elixir-lang/elixir/pull/10680#issuecomment-770075143 👍

Reply:
This should be fixed now.